Emitron/Emitron/UI/Library/LibraryView.swift

```diff
@@ -104,12 +105,17 @@ struct LibraryView: View {
         HStack {
           Image("sort")
             .foregroundColor(.textButtonText)
-
-          Text(filters.sortFilter.name)
-            .font(.uiLabelBold)
-            .foregroundColor(.textButtonText)
+          if [.loading, .loadingAdditional].contains(libraryRepository.state) {
+            Text(filters.sortFilter.name)
+              .font(.uiLabel)
+              .foregroundColor(Color.gray)
+          } else {
+            Text(filters.sortFilter.name)
+              .font(.uiLabelBold)
+              .foregroundColor(.textButtonText)
+          }
         }
-      }
+      }.disabled(libraryRepository.state == .loading || libraryRepository.state == .loadingAdditional)
```

Comment:
Should probably be consistent with line 108. So either this, or switch 108 to match this `||` statement, whichever you feel is clearer.

```suggestion
      }.disabled([.loading, .loadingAdditional].contains(libraryRepository.state))
```

Reply:
Sounds good - I've updated the PR with your suggestions. Thanks!